firmware/sys/at_client/at_client.c

```diff
@@ -46,6 +46,7 @@ uart_buffer_t uart_buffer = {
     .count = 0,
 };
 uint8_t flag_response = 0;
+int is_response = 0;
```

Comment:
All these globals variables has to be documented, and place a warning message to avoid conflicts of variables initialization, outside of this module.